### ISSUE TITLE
Validation set contains strange ids

### ISSUE THREAD

**Issue by 17Skye17** [sallymmx/ActionCLIP#23]
Hi, thank you for releasing the code.
After checking the validation file, I found ~180 video ids that can not be found in official val file or some of them can not be decoded, the videos are listed below:
```
tCKnYXne_o
H-Ww0gGDWU
Sam59CH5_o
blRZN-6_ZM
gxLOV_s9wU
Q2OO6q6-iM
I3_52Xh7oU
sSTHZHHp-c
l309dqYR-8
ebcuq_qItc
7tTouR10Qro
6uq-NBo3Bk
D-Fa71ta14
ioNctElzaas
PcOAmaZMNZY
4SJ-uWc3PQ
WI7e5-wURs
JN0MXb-zi8
IyR-sGt0uw
UqI-TBQRgg
mMT4Nt_c-A
UxN_uuEZC0
6k2ntyDP-c
u8zlG-OS_E
8IjJv90K-M
fZ0IO-Q-ZQ
rrtLyJs-3w
06oD_bFxOQ
Lo3hFbum_o
a06xpsZj-U
5EvC-g-KUQ
GQ-4QfVpXc
TyWiE-4zpM
esH_aGzBrw
7mJ12n-xyM
LktSuL8_7M
Xfmjzt_n24
ctWolbJDJyc
H8Ny92IEyaM
OnU1Hr_jlY
7TZOYU_Ta0
2DwBhMUH-I
9ILBd-ArtM
LKhtbW5q-c
0-s1eu4sF4
jToAVyxs-g
clhd73_vDQ
LESFP2wh_8
NtUqv_6vdU
I02uj1Sc7TM
I0luMKjIZyg
5vTJ-N4jrI
hR-iDJcjgU
F-aEPmjERo
ITTI-fkvo
C84yBh-fQw
NMaC-IGv_Q
YGwB8HJj-g
SxAU4_1c_o
S6wwANH-EY
hu_Ld-ddk
blq_c14hGQ
isSe2P8T-4
owWHGvn_b0
OoJW-OeFtw
G_lySaTeNM
XH_50Lp8qQ
PBf_Wa6vO8
pUDgyU_KGc
DYPEKYAcEFg
HWxZHHT_l4
IAo-mNduUk
6Xp_ymM0Lg
ZY_EfSlzGA
Zpj4-Z2YRk
c3-qJC_azg
B_pr_4s7vY
sAA809R_u1E
6c-sV_gmq0
QfuO07EqYhI
D6-UmndVJk
eJVHxmkm-4
bVVs-nntQ
RZXH93_XNY
5-dvLrzE78
kZsdc1A_J8
54Bs-0kdhA
Y-fUYGcb7o
5g0IDBneA
yAlJ1P_SGg
AgJx-0yaFQ
P56BlJO-gw
J-8cbYBG7c
aQR-rCWaVQ
7qK_w-g3Y
0IEt9-NeV4
TL-9g8KBFA
PB0FuE-fdA
7Od7A1-B9s
egPJubR-CE
Etym1-30wM
3E3GBXAUc
j3eNzQR-EI
rQuS0w-1b8
JcZ7Ry_9kg
j4Anoe2ug8k
Eq_X-uRNm8
0ML-FXomBw
G2XYLk0-38
RlFMUo-JE
QbBRu7a_xM
CMel_KnSzw
W9AQZ-gUro
W07v4Ci-zY
5sx6NEtkd1E
jToBK-njO8
spJJybwq-g
wvsuK9HBif0
r5c12Eo_jY
FawHl2-DAg
dguKqz_F5w
GBKRR-OvqM
adU_0hUdr8
DWE7WQkBvBc
ieIssRi8iXU
Q_vnBY8YP8
afwq-zVgJk
JP5gc9_J4I
iox_MbwxNg
aGmWWA-h3s
vI8Vp2-gfiU
7-dud_cqq8
o9-ONbnlRw
aVXC13LEJgU
MPqy00mB-Y
d8_H5d2sd4
2C-yeMmge0
FGj7-Cxu_0
3FihEVl-R8
cMG2QyN-mE
UZLHav3t_NQ
t_T_nYKdh8
DNb_6w2cZM
D5-ZGEjiWI
58I5s_qDVU
gmBW-mkRXA
VP-VaZhno0
p1Cftd_xo
73Kg-MKmwE
k0w_3JFfmE
m-YKP0ReEE
pn5NxJmok
c96bD-9fHs
jR-X0LqwpY
96AfwOj-qw
LrBC1_yf04
Oni-SybW0
6yaNVdS-2E
4J-bkpjVb4
v7DhQiuKEd0
n0WAbM8z-o
m-2ka9iN9M
WRh2_MJLLs
3MhOA-vSO8
u5A74I0-M
kahgmRD-4g
b-YkpzFphk
vYfm8bO-TM
gtC_avp2gU
5Vu8HJ_eMg
wdnasc-fCg
xxBx1jv-ks
Ykfa-4qx0
UHRaVrN_us
JwMoMeZ_v8
0ew-c0w7uc
uVv1h-xAe0
u4SSk4kWqLA
2L8B_meOLE
c-YbuFrXbI
Bu-6oESyxQ
```
I am not sure if the difference would cause ~1% performance drop in my testing case, since I loaded pretrained model and checked all testing parameters provided in the config file.

**Comment by dreamerlin** [comment]
@17Skye17 It seems that the author will remove '-' and '_' chars when a video id starts with these two.

And @sallymmx Could you please provide the validation videos ?

**Comment by sallymmx** [comment]
您好，我是王蒙蒙，您的邮件我已收到，并及时回复。

**Comment by dreamerlin** [comment]
> 您好，我是王蒙蒙，您的邮件我已收到，并及时回复。

谢谢，请问您能提供这些 video_id 的视频吗，可以发到 jintaolin@smail.nju.edu.cn

```
NtUqv_6vdU
3MhOA-vSO8
JwMoMeZ_v8
LrBC1_yf04
c96bD-9fHs
blq_c14hGQ
bVVs-nntQ
sSTHZHHp-c
WI7e5-wURs
iox_MbwxNg
IAo-mNduUk
2L8B_meOLE
wdnasc-fCg
NMaC-IGv_Q
OoJW-OeFtw
jR-X0LqwpY
c-YbuFrXbI
H-Ww0gGDWU
Xfmjzt_n24
I3_52Xh7oU
UqI-TBQRgg
RlFMUo-JE
Ykfa-4qx0
dguKqz_F5w
2DwBhMUH-I
u8zlG-OS_E
afwq-zVgJk
96AfwOj-qw
4SJ-uWc3PQ
esH_aGzBrw
xxBx1jv-ks
Zpj4-Z2YRk
HWxZHHT_l4
G_lySaTeNM
pUDgyU_KGc
GQ-4QfVpXc
D-Fa71ta14
u5A74I0-M
RZXH93_XNY
d8_H5d2sd4
Q_vnBY8YP8
7TZOYU_Ta0
fZ0IO-Q-ZQ
FawHl2-DAg
t_T_nYKdh8
7Od7A1-B9s
rQuS0w-1b8
n0WAbM8z-o
spJJybwq-g
OnU1Hr_jlY
LKhtbW5q-c
kUfOBKJZczk
vYfm8bO-TM
J-8cbYBG7c
a06xpsZj-U
uVv1h-xAe0
B_pr_4s7vY
YGwB8HJj-g
VP-VaZhno0
r5c12Eo_jY
yAlJ1P_SGg
ITTI-fkvo
GBKRR-OvqM
5Vu8HJ_eMg
D5-ZGEjiWI
06oD_bFxOQ
clhd73_vDQ
IyR-sGt0uw
hR-iDJcjgU
blRZN-6_ZM
G2XYLk0-38
FGj7-Cxu_0
LktSuL8_7M
S6wwANH-EY
gxLOV_s9wU
WRh2_MJLLs
UxN_uuEZC0
TyWiE-4zpM
rrtLyJs-3w
TL-9g8KBFA
5vTJ-N4jrI
HUiafDe-xTo
Etym1-30wM
4J-bkpjVb4
7-dud_cqq8
PBf_Wa6vO8
5EvC-g-KUQ
```

**Comment by 17Skye17** [comment]
> @17Skye17 It seems that the author will remove '-' and '_' chars when a video id starts with these two.
> 
> And @sallymmx Could you please provide the validation videos ?

yes, it is, after training the model with my own dataset copy I still can't get the reported accuracy, so I was wondering if the validation set is a devil.

**Comment by sallymmx** [comment]
The validation videos are set like the official K400.
